.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -11,10 +11,13 @@ assignees: ''
 - [ ] Update changelog if necessary, and merge a PR for it to main
```

Comment:
I think this was moved to later in the process. Would moving it before this step `Using the same git sha, create a calver tag for the FULL RELEASE, example format `24.4.0` make sense?